core/src/test/java/com/redhat/lightblue/client/expression/query/ValueQueryTest.java

```diff
@@ -33,4 +35,25 @@ public void testToStringCallsToJson() throws JSONException {
 
         JSONAssert.assertEquals(expression.toString(), expression.toJson(), false);
     }
+
+    @Test
+    public void testValueWithWhiteSpace() throws JSONException {
+        ValueQuery expression = new ValueQuery("field1 = Red Hat Enterprise Linux");
+
+        String expectedJson = "{\"field\":\"field1\",\"op\":\"=\",\"rvalue\":\"Red Hat Enterprise Linux\"}";
+
+        JSONAssert.assertEquals(expectedJson, expression.toJson(), false);
+    }
+
+    @Test(expected=IllegalArgumentException.class)
+    public void testInvalidExpression() {
+        new ValueQuery("fie ld1 = Red Hat Enterprise Linux");
+        Assert.fail();
+    }
+
+    @Test(expected=IllegalArgumentException.class)
+    public void testInvalidOperator() {
+        new ValueQuery("field1 ~= Red Hat Enterprise Linux");
+        Assert.fail();
```

Comment:
Assert.fail() is not needed as it is already expecting and IllegalArgumentException. If no exception, or the wrong exception is thrown, junit will fail the test.